CHANGELOG.rst

```diff
@@ -4,6 +4,61 @@ CHANGELOG
 This document describes changes between each past release as well as
 the version control of each dependency.
 
+10.0.0 (2018-09-12)
+===================
+
+kinto
+'''''
+
+**kinto 9.2.3 → 10.0.0**: https://github.com/Kinto/kinto/releases/tag/10.0.0
+
+**Breaking changes**
+
+- ``kinto.core.events.get_resource_events`` now returns a generator
+  rather than a list.
+
+
+**New features**
+
+- Include Python 3.7 support.
+- ``kinto.core.events.notify_resource_event`` now supports
+  ``resource_name`` and ``resource_data``. These are useful when
+  emitting events from one view "as though" they came from another
+  view.
+- Resource events can now trigger other resource events, which are
+  handled correctly. This might be handy if one resource wants to
+  simulate events on another "virtual" resource, as in ``kinto-changes``.
+
+**Bug fixes**
+
+- Raise a configuration error if the ``kinto.plugin.accounts`` is included without being enabled in policies.
+  Without this *kinto-admin* would present a confusing login experience (fixes #1734).
+
+**Internal changes**
+
+- Upgrade kinto-admin to v1.20.0
```

Comment:
I didn't update the kinto-admin version -- this is just from the release notes in the Kinto changelog. Probably I should have updated it as part of the kinto release. I could do that now but since the kinto-admin changes are mostly dependency updates, I don't know if it's really worth it.

Reply:
It turned out that kinto-admin was actually at version v1.20.1 in this release.